### ISSUE TITLE
Show wallet data is stale on UI

### ISSUE THREAD

**Issue by cjackie** [cjackie/OpenWallet#16]
### Problem
After we send BTC, the wallet does not show the TX is in the pending TX, until we reload the `viewwallet` page. As a result, the user need to keep refreshing until they know the TX is pending. 

### Proposal
Expose a API on the backend that given a wallet ID and returns UTC timestamp of last update time when the wallet data is changed (e.g. pending TX added, or UTXO removed). The frontend periodically calls the API and compare with the last updated time it has, when the last updated time is greater than the current one, the frontend display a reload button and indicate the data is stale.

In summary, basically, the backend exposes some data letting the frontend knows the wallet is stale.
